src/services/user.service.js

```diff
@@ -34,7 +35,54 @@ const getById = async (id) => {
   }
 };
 
+const getUsers = async (programRef) => {
+  try {
+    const userCollection = collection(db, 'users-v2');
+    
+    if (programRef) {
+      const q = query(userCollection, where('appliciant_data.programRef', '==', programRef));
+      const userSnapshot = await getDocs(q);
+      const users = userSnapshot.docs.map(doc => ({ id: doc.id, ...doc.data() }));
+      return users;
+    } else {
+      const userSnapshot = await getDocs(userCollection);
+      const users = userSnapshot.docs.map(doc => ({ id: doc.id, ...doc.data() }));
+      return users;
```

Comment:
You have code repetition. lines 45-46 & 49-50.
move the lines outside the if-else block.